INSTALL

```diff
@@ -7,13 +7,12 @@ Something like:
 --------------------
 $ git clone git://github.com/tricycle/predictionbook.git
 $ cd predictionbook
-$ cp config/database.yml.example config/database.yml # update db user / password / database names if necessary
-$ cp config/credentials.yml.example config/credentials.yml
-$ bundle install
-$ rake secret # run twice ; set 'session_secret' / 'rest_auth_site_key' in credentials.yml
-$ rake db:create
-$ rake db:schema:load # or db:migrate and db:test:prepare
-$ rake spec # run specs
+$ cp config/database.yml.sample config/database.yml # update db user / password / database names if necessary
+$ cp config/credentials.yml.sample config/credentials.yml
+$ bin/setup
+$ rails secret # run twice ; set 'session_secret' / 'rest_auth_site_key' in credentials.yml
+$ rails db:schema:load # or db:migrate and db:test:prepare
```

Comment:
Hrm is this line strictly necessary? I thought that RSpec runs `db:test:prepare` which does a `schema:load` under the hood before running tests.

Reply:
It isn't necessary. I switched it out to `rails db:migrate` [here](https://github.com/bellroy/predictionbook/pull/186/commits/3d340acdb935e874d4375b172ad8a0cc6e18fe9c). IMO, `bin/setup` should run migrations, but I was hoping to mostly fix the incorrect file names in the install doc, since it just won't work as written, but maybe there's a reason we don't want that in the setup script. 🤷🏼‍♂️ 

Also, please squash first if you end up merging. 👍🏼